src/test/java/seedu/address/logic/commands/DeleteTagCommandTest.java

```diff
@@ -71,33 +74,32 @@ public void execute_multipleExistingTags_success() {
      * EP: Single non-existent tag.
      */
     @Test
-    public void execute_nonExistentTag_successWithWarning() {
+    public void executeNonExistentTagWithError() {
         Tag nonExistentTag = BRIDES_SIDE;
         List<Tag> nonExistentTags = List.of(nonExistentTag);
 
-        DeleteTagCommand newTagCommand = new DeleteTagCommand(nonExistentTags);
-        String expectedMessage = DeleteTagCommand.MESSAGE_NONEXISTENT;
+        DeleteTagCommand newTagCommand = new DeleteTagCommand(nonExistentTags, false);
+        String expectedMessage = BRIDES_SIDE + " " + DeleteTagCommand.MESSAGE_NONEXISTENT;
 
         Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
 
-        // No exception is thrown, but the user will be notified of
-        // non-existent tag(s).
         assertCommandSuccess(newTagCommand, model, expectedMessage, expectedModel);
     }
 
     /**
      * EP: A mix of existing and non-existent tags.
      */
     @Test
-    public void execute_mixedExistingAndNonExistentTags_successWithWarning() {
+    public void execute_mixedExistingAndNonExistentTags_withError() {
         Tag existingTag = TypicalTags.BRIDES_SIDE;
         Tag newColleaguesTag = TypicalTags.COLLEAGUES;
         Tag newFriendsTag = TypicalTags.FRIENDS;
         List<Tag> mixedTags = List.of(newFriendsTag, existingTag, newColleaguesTag);
         model.addTag(existingTag);
 
-        DeleteTagCommand newTagCommand = new DeleteTagCommand(mixedTags);
-        String expectedMessage = DeleteTagCommand.MESSAGE_NONEXISTENT;
+        DeleteTagCommand newTagCommand = new DeleteTagCommand(mixedTags, false);
+        String expectedMessage = DeleteTagCommand.MESSAGE_SUCCESS + " " + FRIENDS + " " + COLLEAGUES
```

Comment:
Maybe the toString() method of Lists can be used instead of manually appending a space to each tag